src/support/steps/performance-hints.ts

```diff
@@ -0,0 +1,275 @@
+/**
+ * @fileoverview
+ * This module contains Cucumber step definitions using Playwright for asserting performance hints data clearing from DB, 
+ * both ATF and LRC tables
+ * 
+ * @requires {@link ../../common/custom-world}
+ * @requires {@link @playwright/test}
+ * @requires {@link @cucumber/cucumber}
+ */
+
+import { ICustomWorld } from "../../common/custom-world";
+import { WP_BASE_URL } from '../../../config/wp.config';
+import { When, Then } from '@cucumber/cucumber';
+import { dbQuery, getWPTablePrefix } from "../../../utils/commands";
+import { extractFromStdout } from "../../../utils/helpers";
+import { expect } from "@playwright/test";
+
+function delay(ms: number) {
+    return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+/*
+ * Executes step to add hardcoded data to DB: ATF & LRC tables
+ */ 
+When('performance hints data added to DB', async function (this: ICustomWorld) {
+    const tablePrefix = await getWPTablePrefix();
```

Comment:
@hanna-meda @jeawhanlee what do you think about having utility to seed data in DB and be called here?